modules/onscreengui/src/renderproperties.cpp

```diff
@@ -46,13 +46,22 @@ void renderTooltip(Property* prop) {
     }
 }
 
-void executeScript(const std::string& id, const std::string& value) {
+void executeScriptSingle(const std::string& id, const std::string& value) {
```

Comment:
Agreed! I just wanted to check if you meant to use the method somewhere in this pull request. Right now, all calls are made to the `executeScript` (alias for `executeScriptGroup`) (as far as I can see)